public/static/locales/en/listings.json

```diff
@@ -59,7 +59,7 @@
     "main": "Find travel insurance providers if you have a serious medical condition",
     "filter": "Refine your search",
     "sub_filter": "Answer these questions to help you narrow down the list of firms to choose from",
-    "clear": "Clear",
+    "clear": "Clear filter",
```

Comment:
This should be "Clear filters" (plural).

Reply:
REsolved

Reply:
> > I'm aware this change wasn't strictly part of this work but I think we need to look at an alternative way of breaking the title. By adding a `<br>` tag to the text we are causing awkward alignments on some viewport widths, such as below. Also there is no equivalent in the Welsh text as it stands.
> > ![image](https://user-images.githubusercontent.com/6080548/93187630-e9714d00-f737-11ea-8ea4-c255c362176b.png)
> 
> We can set a definite width causing it to break responsively and it will naturally be applied to welsh when the translation is available but it has to be cleared with the Product owner/manage QA because it might not break at the exact point they want but it will make sense.

Discussed this with Dobrin and he agreed we should do what we deem best. So i have set a width to it.

Reply:
OK, that looks good to me 👍

Reply:
> REsolved

👍